pkg/provider/gitlab.go

```diff
@@ -47,14 +49,17 @@ func (repo *GitLabRepository) Init(config map[string]string) error {
 		return fmt.Errorf("gitlab_projectid is required")
 	}
 
+	var err error
+	stripVTagPrefix := config["strip_v_tag_prefix"]
+	repo.stripVTagPrefix, err = strconv.ParseBool(stripVTagPrefix)
```

Comment:
this will cause an error if `strip_v_tag_prefix` is not set, so we should check before if the config value is not an empty string

Reply:
If the config is not set, the returned value is an empty string since it's a `map[string]string` and then if we hit an error the returned boolean is `false`.

Reply:
I don't understand. The following lines return with an error instead of setting it to false 🤔

Reply:
My bad, misunderstood the point, I'll fix that

Reply:
Done

Reply:
thank you ☺️